### PR TITLE
DOC: Fixed table formatting in box plot section

### PR DESCRIPTION
**Pull request by pvanhauw**
[DOC]fixed table formatting in the box plot section keeing simple rst table format

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/user_guide/visualization.rst [hunk 1]
```diff
@@ -444,8 +444,7 @@ keyword, will affect the output type as well:
 
 ================ ======= ==========================
 ``return_type=`` Faceted Output type
```
Comment (MarcoGorelli) [inline doc/source/user_guide/visualization.rst:446]: I think we can remove the `=` - I know it was there originally, but I don't think it really makes sense

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
Thanks @pvanhauw , it look better like this!

**Comment by MarcoGorelli** [comment]
Also @pvanhauw can you update the title of the PR to `[DOC]fixed table formatting in the box plot section`?

**Comment by pvanhauw** [comment]
> Also @pvanhauw can you update the title of the PR to `[DOC]fixed table formatting in the box plot section`?

I  just did

**Comment by jorisvandenbossche** [comment]
@pvanhauw it seems the PR title rename didn't come through

**Review by jorisvandenbossche — APPROVED** [review]
Thanks for the fix!

**Comment by pvanhauw** [comment]
thanks for helping me on my first PR !

**Pull request merged**